Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -133,6 +133,21 @@ bool CMainConfig::Load()
         return false;
     }
 
+    // Grab rules
+    CXMLNode* pNode = nullptr;
+    std::size_t uiCurrentIndex = 0;
+    while (pNode = m_pRootNode->FindSubNode("rule", uiCurrentIndex++))
+    {
+        CXMLAttribute* pAttribute = pNode->GetAttributes().Find("name");
+        SString        strName = pAttribute ? pAttribute->GetValue() : SString{};
+
+        pAttribute = pNode->GetAttributes().Find("value");
+        SString strValue = pAttribute ? pAttribute->GetValue() : SString{};
+
+        if (!strName.empty() && !strValue.empty())
+            m_RulesForASEMap[std::move(strName)] = std::move(strValue);
```

Comment:
if you can, please dont use hungarian notation

Reply:
Only here? The rest of the entire file is all hungarian

Reply:
If you could, you can make a small refactor there.

Reply:
Ok I'll try my best.